pandas/tests/io/formats/test_style.py

```diff
@@ -891,6 +891,116 @@ def test_mi_sparse_column_names(self):
         ]
         assert head == expected
 
+    def test_hide_single_index(self):
```

Comment:
For this test all of other tests that you've added, reference your PR (or issue number) under the function definition.

Reply:
thanks!